website/_config.yml

```diff
@@ -0,0 +1,83 @@
+# Hexo Configuration
+## Docs: https://hexo.io/docs/configuration.html
+## Source: https://github.com/hexojs/hexo/
+
+# Site
+title: Hexo
+subtitle:
+description:
+author: John Doe
+language:
+timezone:
+
+# URL
+## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
+url: https://github.com/OpenGeoscience/geojs
+root: /geojs/
+permalink: :year/:month/:day/:title/
+permalink_defaults:
+
+# Directory
+source_dir: source
+public_dir: public
+tag_dir: tags
+archive_dir: archives
+category_dir: categories
+code_dir: downloads/code
+i18n_dir: :lang
+skip_render:
+  - "**/*.js"
+  - "data/**/*"
+  - "apidocs/**/*"
+
+# Writing
+new_post_name: :title.md # File name of new posts
+default_layout: layout
+titlecase: false # Transform title into titlecase
+external_link: true # Open external links in new tab
+filename_case: 0
+render_drafts: false
+post_asset_folder: false
+relative_link: false
```

Comment:
Can we set this to `true`?  Doing so will also require changing `website/themes/agency/layout/layout.ejs` (see comment there).  By using relative links, you can use any webserver to test the website, not just use `haxo server`, and, if we deploy to a sample to a proxied container, the website will still work.